llvm/utils/UpdateTestChecks/common.py

```diff
@@ -650,6 +655,16 @@ def get_triple_from_march(march):
     print("Cannot find a triple. Assume 'x86'", file=sys.stderr)
     return "x86"
 
+def get_global_underscores(raw_tool_output):
+    m = DATA_LAYOUT_RE.search(raw_tool_output)
+    if not m:
+        return False
+    data_layout = m.group("layout")
+    idx = data_layout.find("m:")
+    if idx < 0:
+        return False
+    ch = data_layout[idx + 2]
+    return ch == 'o' or ch == 'x'
```

Comment:
I originally added the mangled name parsing to support macOS, which mach-o symbols does it not work for?

Reply:
See the `arm_neon_intrinsics.c` test that I regenerated in this patch. It's also intended to serve as a test of the patch itself.

Reply:
I've now removed the changes to `arm_neon_intrinsics.c`